### PR TITLE
Hello Microverse Project

### PR DESCRIPTION
**Pull request by Rachelwebdev**
## Set up a "Hello Microverse" Project

- Added a gitignore file
- Added node modules to the gitignore file
- Set up linter for HTML and CSS
- Added a descriptive README file
- Added .HTML file
- Added .css file
- Added test.md file and put it in gitignore

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,86 @@
...
+💻 Getting Started
+Setup
+Prerequisites
+Install
+Usage
+👥 Authors
+🤝 Contributing
+🙏 Acknowledgements
+
+📝 License
```
Comment (Gambit142) [inline README.md]: - [ ] Your README.md file should be professional at all times and they are mostly the first point of contact with your project when people want to assess your project. The highlighted section should be linked to the appropriate part in the README.md file just the way we have them in the original _**README.md**_ template. Kindly add or remove them if they can't be linked properly. 😄 👍

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @Rachelwebdev 👋, 

Good job so far!

## Highlights
- All linters checks passed ✔️ 
- Used correct Github flow ✔️ 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] Your commit messages are meaningful but they do not follow the convention of being capitalized. For example, we can have:
- `Add the stylesheet link to the css file` and not `add the stylesheet link to the css file`. Notice the capitalized 'A'. You can apply this in future projects 👍 
- [ ] Kindly add a descriptive pull request summary. A pull request summary should explain changes that will be introduced to the project through the branch you created.

- _Kindly check the comments under the review for further changes._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👏👏

> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md (hunk 1 above)
Comment (Rachelwebdev) [reply README.md]: Thank you @Gambit142  for the feedback. I would work on these changes as you have highlighted

**Review by mgmediaweb — APPROVED** [review]
Hi @Rachelwebdev , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approved5

## Highlights 💯 
- Good job following the advice of the previous reviewer ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**